### PR TITLE
TOTP and b64 config parsing

### PR DESCRIPTION
**Pull request by idimov-keeper**
Added TOTP parsing

### CODE REVIEW

File: sdk/java/core/src/main/kotlin/com/keepersecurity/secretsManager/core/CryptoUtils.kt [hunk 1]
```diff
@@ -13,6 +16,8 @@ import javax.crypto.KeyAgreement
 import javax.crypto.Mac
 import javax.crypto.spec.GCMParameterSpec
 import javax.crypto.spec.SecretKeySpec
+import kotlin.experimental.and
```
Comment (maksimu) [inline sdk/java/core/src/main/kotlin/com/keepersecurity/secretsManager/core/CryptoUtils.kt]: What does `kotlin.experimental` mean? Maybe it should not be used in production?

File: sdk/dotNet/SecretsManager/SecretsManager.csproj [hunk 2]
```diff
@@ -27,4 +27,7 @@
       <PackageReference Include="System.Text.Encodings.Web" Version="5.0.1" />
     </ItemGroup>
 
+    <ItemGroup>
```
Comment (saldoukhov) [inline sdk/dotNet/SecretsManager/SecretsManager.csproj]: This should not be necessary

File: sdk/java/core/src/main/kotlin/com/keepersecurity/secretsManager/core/SecretsManager.kt [hunk 3]
```diff
@@ -9,11 +9,18 @@ import kotlinx.serialization.encodeToString
 import kotlinx.serialization.json.*
 import java.net.HttpURLConnection.HTTP_OK
 import java.net.URL
+import java.net.URLDecoder
```
Comment (saldoukhov) [inline sdk/java/core/src/main/kotlin/com/keepersecurity/secretsManager/core/SecretsManager.kt]: This should not be necessary

File: sdk/javascript/packages/core/src/platform.ts [hunk 4]
```diff
@@ -1,3 +1,5 @@
+import { createHash, createHmac, Hmac } from "crypto"
```
Comment (saldoukhov) [inline sdk/javascript/packages/core/src/platform.ts]: This is still not fixed

**Pull request merged**